maryana_khoptyak/Task_1/func_get_max_value.py

```diff
@@ -0,0 +1,10 @@
+
+def get_max_value(data):
+    """ Function which returns a maximum integer value from list of strings """
+    sorted_nums_list = []
+    for value in data:
+        try:
+            sorted_nums_list.append(float(value))
+        except ValueError:
+            pass
```

Comment:
It's not very good idea to have a `pass` statement in code. 
So, you should replace it with some print, etc.

Reply:
Pass statement is replaced.